recipe/bld.bat

```diff
@@ -1,50 +1,185 @@
 @echo On
+setlocal enabledelayedexpansion
+
+REM remove pyproject.toml to avoid installing deps from pip
+DEL pyproject.toml
 
 set TH_BINARY_BUILD=1
 set PYTORCH_BUILD_VERSION=%PKG_VERSION%
 set PYTORCH_BUILD_NUMBER=%PKG_BUILDNUM%
 
-if "%pytorch_variant%" == "gpu" (
-    set build_with_cuda=1
-    set desired_cuda=%CUDA_VERSION:~0,-1%.%CUDA_VERSION:~-1,1%
+REM I don't know where this folder comes from, but it's interfering with the build in osx-64
+RD /S /Q %PREFIX%\git
+
+@REM Setup BLAS
+if "%blas_impl%" == "generic" (
+    REM Fake openblas
+    SET BLAS=OpenBLAS
+
+    sed -i.bak "s#FIND_LIBRARY.*#set(OpenBLAS_LIB %PREFIX:\=/%/Library/lib/lapack.lib %PREFIX:\=/%/Library/lib/cblas.lib %PREFIX:\=/%/Library/lib/blas.lib)#g" cmake/Modules/FindOpenBLAS.cmake
```

Comment:
how many occurances are there? can we write a small patch and avoid the `m2-sed`? `m2` pulls in many strange libraries that can confuse the compilers.

Reply:
> `m2` pulls in many strange libraries that can confuse the compilers.

This shouldn't be a big issue anymore AFAIU. But yes, patch is cleaner than a sed script, so overall +0.5 to @hmaarrfk's suggestion

Reply:
I can most likely replace this invocation with a patch. Note that I just took this script from the non windows build. I will replace the invocation there too.

Sed is also called a few lines below to support megabuild. Im not sure how to use patch for that, so I think I will leave the `m2-sed` dependency there.

```batch
sed "s/3.12/%PY_VER%/g" build/CMakeCache.txt.orig > build/CMakeCache.txt
sed -i "s/312/%CONDA_PY%/g" build/CMakeCache.txt
```

Reply:
I replaced the `sed` command with a patch that does some processing in CMake instead.